### ISSUE TITLE
annoying gnome smartcard dialog is opened on each app launch

### ISSUE THREAD

**Issue by MartinLoeper** [Yubico/yubioath-flutter#1742]
Every time I open the app I am asked by gnome: "Please insert card with serial number <yubikey id>"

I cannot click "Ok" and have to click "Cancel" to proceed.
I wonder why this is happening. Is the app trying to spawn a process like "scdaemon" to interact with the smartcard?
This could be as well a libsecret issue because my libsecret backend uses gpg. 

Is there an option to increase log verbosity to help track this down?

**Comment by dainnilsson** [comment]
I would guess it's libsecret then. This project uses https://github.com/jaraco/keyring to store the OATH access key if you've selected "Remember password" when unlocking the `Accounts` view. Does that sound like a likely culprit to you? You may be able to configure keyring to use a different backend, or by not opting to remember any passwords for you.

**Comment by MartinLoeper** [comment]
Thanks @dainnilsson. I figured out that ykman and yubioath-flutter somehow do not play nicely together with [pass-secret-service](https://github.com/nix-community/home-manager/blob/master/modules/services/pass-secret-service.nix). I switched back to gnome-keyring and it works as expected again.

Unfortunately, I do not have time to investigate this incompatibility. I just switched my libsecret backend. Maybe someone else who reads this has an idea...
